backend/app/processing/vanna_integrator.py

```
@@ -0,0 +1,267 @@
+                
+            # Train on first table
+            conn = sqlite3.connect(db_path)
+            cursor = conn.cursor()
+            cursor.execute("SELECT name FROM sqlite_master WHERE type='table' LIMIT 1;")
+            table = cursor.fetchone()
+            if table:
+                # Train the model
+                self.vn.train(f"SELECT * FROM {table[0]};")
+                result["trained_on"] = table[0]
+            conn.close()
+        except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)